src/modules/Snapper.rb

```diff
@@ -149,7 +149,7 @@ def GetFileModification(file, old, new)
         # the file diff
         elsif Ops.get(out, "stdout") != ""
           status = ["diff"]
-          ret["diff"] = out["stdout"].encode(Encoding::UTF_8, { :invalid => :replace })
+          ret["diff"] = out["stdout"].encode(Encoding::UTF_8, invalid: :replace) # rubocop:disable Style/HashSyntax
```

Comment:
well, do you try if `:invalid => :replace` can be used to be consistent with rest of modules ( even if it is inconsistent with rest of YaST ) ?

Reply:
> well, do you try if `:invalid => :replace` can be used to be consistent with rest of modules

I'll try it.

> ( even if it is inconsistent with rest of YaST ) ?

I did search for it before making the PR. And  I found it only two similar calls

https://github.com/yast/yast-ruby-bindings/blob/f634ea70c3c48cc0d89a87031d9a3e77377d40f1/src/ruby/yast/logger.rb#L28-L33

and

https://github.com/yast/yast-yast2/blob/ad8d240b5e6a76773dc4a367aa02b9ba9feb3457/library/packages/src/modules/PackageCallbacks.rb#L491-L495